src/worker.cpp

```diff
@@ -1466,4 +1466,90 @@ void Worker::performBackupAndRestore(types::VPDMapVariant& io_srcVpdMap)
         */
     }
 }
+
+void Worker::deleteFruVpd(const std::string& i_dbusObjPath)
+{
+    try
+    {
+        const bool l_presentPropValue =
```

Comment:
should we check i_dbusObjPath is empty or not ?

Reply:
Resolving as done.